backend/src/main/java/com/google/sps/data/TimeHelper.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+import java.time.ZonedDateTime;
+
+/**
+ * Class used to set reference date/time for testing and convert input timeAvailableUntil to date
+ * timestamp
+ */
+public final class TimeHelper {
+
+  private static ZonedDateTime dateTime;
+
+  /** Constructor */
+  public TimeHelper() {
+    this(ZonedDateTime.now());
+  }
```

Comment:
Sorry I missed this earlier. I'd recommend eliminating this constructor and only using the one below. When classes construct their own dependencies (e.g. this class constructing its ZoneDateTime internally here in the constructor) they become harder to test. Using "dependency injection" as you have in the constructor below allows a class' dependencies to be provided from the outside which in turn makes them easier to test.

 Since you don't want to test one code path (the constructor that accepts a ZoneDateTime) while using another in prod, I would recommend simplifying down to one.